src/vitabel/vitals.py

```diff
@@ -1156,30 +1158,41 @@ def shocks(self):
 
     def compute_etco2_and_ventilations(
         self,
-        mode="threshold",
-        breaththresh: float = 4,
-        etco2_thresh: float = 4,
+        mode: Literal['threshold', 'filter'] = 'filter',
+        breath_thresh: float = 2,
+        etco2_thresh: float = 3,
+        **kwargs,
     ):
-        """Computes the etCO2 values and the timestamps of the ventilations based
-        on CO2 waveform and add it to the labels.
+        """
+        Computes end-tidal CO2 (etCO₂) values and timestamps of ventilations from the capnography waveform,
+        and adds them as labels.
 
-        The capnography signal must be a channel of the file and needs to be named 'capnography'. Currently two different versions are implemented: 'threshold' and 'filter'. The default is 'filter'.
+        The capnography signal must be present as a channel named 'capnography'. Two detection methods are supported:
```

Comment:
ReST requires a blank line before an itemization

```suggestion
        The capnography signal must be present as a channel named 'capnography'. Two detection methods are supported:

```